src/com/cft/contactmerge/SupportedFileType.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * List of file types supported for import and export
+ */
+package com.cft.contactmerge;
+
+public enum SupportedFileType {
+    XML,
+    TSV,
+    CSV
```

Comment:
we should add XLS and XLSX

Reply:
As per our conversation last week... I don't see us actually supporting those in the near term but once we do, yes, we should add. ITK doesn't require them as Excel works well with TSV and TSV is easier to implement and is more universal.